docs/tutorial/deployments-ux.md

```diff
@@ -1,5 +1,5 @@
 ---
-description: Learn how Prefect projects allow you to easily manage your code and deployments.
```

Comment:
We probably want to change the title and update the first two paragraphs of this page.

Reply:
I've updated the tutorial page to not include references to projects 👍